paper-input-behavior.html

```diff
@@ -356,6 +356,15 @@
 
     attached: function() {
       this._updateAriaLabelledBy();
+
+      // These types have some default placeholder text; overlapping
+      // the label on top of it looks terrible. Auto-float the label in this case.
+      var typesThatHaveText = ["date", "datetime", "datetime-local", "month",
+          "search", "file", "time", "week"];
```

Comment:
This seems worth storing in a static so that we don't create an array for every time every input is attached. :sake: